packages/interactive_media_ads/lib/src/ad_display_container.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+
+import 'platform_interface/platform_ad_display_container.dart';
+import 'platform_interface/platform_interface.dart';
+
+/// Handles playing ads after they've been received from the server.
+///
+/// ## Platform-Specific Features
+/// This class contains an underlying implementation provided by the current
+/// platform. Once a platform implementation is imported, the examples below
+/// can be followed to use features provided by a platform's implementation.
+///
+/// {@macro interactive_media_ads.AdDisplayContainer.fromPlatformCreationParams}
+///
+/// Below is an example of accessing the platform-specific implementation for
+/// iOS and Android:
+///
+/// ```dart
+/// final AdDisplayContainer container = AdDisplayContainer();
+///
+/// if (InteractiveMediaAdsPlatform.instance is IosInteractiveMediaAdsPlatform) {
```

Comment:
`IOS` here, and throughout the PR in similar uses. See https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#capitalize-identifiers-consistent-with-their-spelling